test-data/golden/two_group_securejoins

```diff
@@ -1,9 +1,10 @@
 Group#Chat#11: Group [3 member(s)] 🛡️
 --------------------------------------------------------------------------------
-Msg#11: info (Contact#Contact#Info): alice@example.org invited you to join this group.
+Msg#13: info (Contact#Contact#Info): alice@example.org invited you to join this group.
 
 Waiting for the device of alice@example.org to reply… [NOTICED][INFO]
-Msg#13: info (Contact#Contact#Info): alice@example.org replied, waiting for being added to the group… [NOTICED][INFO]
-Msg#17: info (Contact#Contact#Info): Messages are guaranteed to be end-to-end encrypted from now on. [NOTICED][INFO 🛡️]
-Msg#18🔒:  (Contact#Contact#10): Member Me added by alice@example.org. [FRESH][INFO]
+Msg#15: info (Contact#Contact#Info): alice@example.org replied, waiting for being added to the group… [NOTICED][INFO]
+Msg#12: info (Contact#Contact#Info): Messages are end-to-end encrypted. [NOTICED][INFO]
+Msg#19: info (Contact#Contact#Info): Messages are end-to-end encrypted. [NOTICED][INFO 🛡️]
```

Comment:
User annoyance: It's not that nice that the message is put into the chat twice. Not sure how often this happens.